tests/IceRpc.Quic.Tests/Transports/QuicTransportSslAuthenticationTests.cs

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) ZeroC, Inc.
+
+using IceRpc.Tests.Common;
+using IceRpc.Transports;
+using Microsoft.Extensions.DependencyInjection;
+using NUnit.Framework;
+using System.Net.Quic;
+using System.Net.Security;
+using System.Security.Authentication;
+using System.Security.Cryptography.X509Certificates;
+
+namespace IceRpc.Tests.Transports;
+
+/// <summary>Test Ssl authentication with Quic transport.</summary>
+public class QuicTransportSslAuthenticationConformanceTests
```

Comment:
I would remove "conformance" from the class name. "Conformance" is related to the Multiplexed / Duplex Transport abstraction.